Dockerfile.extra

```diff
@@ -0,0 +1,58 @@
+# -- plugins -------------------------------------------------------------------
+
+FROM development AS plugins
+
+WORKDIR /tmp/vast
+
+RUN cmake -S contrib/vast-plugins/compaction -B build-compaction -G Ninja \
+      -D CMAKE_INSTALL_PREFIX:STRING="$PREFIX" && \
+      cmake --build build-compaction --parallel && \
+      cmake --install build-compaction --strip && \
+      rm -rf build-compaction
+
+RUN cmake -S contrib/vast-plugins/fleet -B build-fleet -G Ninja \
+      -D CMAKE_INSTALL_PREFIX:STRING="$PREFIX" && \
+      cmake --build build-fleet --parallel && \
+      cmake --install build-fleet --strip && \
+      rm -rf build-fleet
+
+RUN cmake -S contrib/vast-plugins/inventory -B build-inventory -G Ninja \
+      -D CMAKE_INSTALL_PREFIX:STRING="$PREFIX" && \
+      cmake --build build-inventory --parallel && \
+      cmake --install build-inventory --strip && \
+      rm -rf build-inventory
+
+RUN cmake -S contrib/vast-plugins/matcher -B build-matcher -G Ninja \
+      -D CMAKE_INSTALL_PREFIX:STRING="$PREFIX" && \
+      cmake --build build-matcher --parallel && \
+      cmake --install build-matcher --strip && \
+      rm -rf build-matcher
+
+RUN cmake -S contrib/vast-plugins/netflow -B build-netflow -G Ninja \
+      -D CMAKE_INSTALL_PREFIX:STRING="$PREFIX" && \
+      cmake --build build-netflow --parallel && \
+      cmake --install build-netflow --strip && \
+      rm -rf build-netflow
+
+# -- vast-ce -------------------------------------------------------------------
+
+FROM production AS vast-ce
+
+COPY --from=plugins --chown=vast:vast $PREFIX/lib/vast/plugins/libvast-plugin-matcher.so $PREFIX/lib/vast/plugins/
+COPY --from=plugins --chown=vast:vast $PREFIX/share/doc/matcher $PREFIX/share/doc/matcher
+COPY --from=plugins --chown=vast:vast $PREFIX/lib/vast/plugins/libvast-plugin-netflow.so $PREFIX/lib/vast/plugins/
+COPY --from=plugins --chown=vast:vast $PREFIX/share/doc/netflow $PREFIX/share/doc/netflow
+
+# -- vast-ee -------------------------------------------------------------------
+
+FROM vast-ce AS vast-ee
```

Comment:
These stages are lacking documentation at https://vast.io/docs/setup/deploy/docker#build-your-own-vast-image

Reply:
They are not usable for anyone without access to private repositories. Adding public docs for them would just give people the wrong impression.